readest: 0.9.88 -> 0.9.91

ddogfoodd opened the pull request:
Changelog: https://github.com/readest/readest/releases/tag/v0.9.91
Diff: https://github.com/readest/readest/compare/v0.9.88...v0.9.91

## Things done
- Built on platform:
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [ ] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [x] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [x] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

ddogfoodd committed:
readest: 0.9.88 -> 0.9.90

Changelog: https://github.com/readest/readest/releases/tag/v0.9.90
Diff: https://github.com/readest/readest/compare/v0.9.88...v0.9.90

Sigmanificient commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 456423`
Commit: [`8f2cc0e16543444f69f079a086279f29aa1862b6`](https://github.com/NixOS/nixpkgs/commit/8f2cc0e16543444f69f079a086279f29aa1862b6) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/8f2cc0e16543444f69f079a086279f29aa1862b6..pull/456423/head))
Merge: [`808a06a9fb4d395ebfa83e6d030654c62e01d60d`](https://github.com/NixOS/nixpkgs/commit/808a06a9fb4d395ebfa83e6d030654c62e01d60d)

Logs: https://github.com/Sigmanificient/nixpkgs-review-gha/actions/runs/18908945632


---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>readest</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>readest</li>
  </ul>
</details>

ddogfoodd committed:
readest: 0.9.90 -> 0.9.91

Changelog: https://github.com/readest/readest/releases/tag/v0.9.91
Diff: https://github.com/readest/readest/compare/v0.9.90...v0.9.91

eljamm commented:
> The changelog pop-up on open still shows 0.9.90 for me. However, about shows 0.9.91.

That's odd. When I opened it, now, it showed both 0.9.90 and 0.9.91.

eljamm reviewed (approved):
Seems good from my tests.
